File: Java/Basics/file6.java

import java.util.*;

public class file6 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int num = sc.nextInt(), result = 0;

        if(num > 20) {
            System.out.println("Wrong Infrastructure..");
        } else {
            result = num + (num/2) + (num % 2);
             System.out.println(result);
        }
       
    }
}

File: C/Basics/file6.c

#include<stdio.h>

int main()
{
    int num, result ;
    scanf("%d", &num);
    if(num > 20) {
        printf("Wrong infrastructure");
    } else {
        result = num + (num/2) + (num % 2);
    }
    printf("%d", result);
}